src/mayara_pi.cpp

```
@@ -1701,6 +1901,58 @@ void mayara_pi::PlayGuardAlarmSound() {
+  }
+
+  const wxString out_path = m_server ? m_server->InstallDir() +
+                                           wxFileName::GetPathSeparator() +
+                                           "changelog.html"
+                                     : wxString();
+  if (out_path.IsEmpty()) {
+    wxMessageBox(_("Nowhere writable was found for the changelog page."),
+                _("Mayara"), wxOK | wxICON_WARNING, parent);
+    return;
+  }
+  wxFFile out(out_path, "wb");
```

Clang (14.0.6)

[warning] variable 'out' is not initialized

(cppcoreguidelines-init-variables)